packages/dd-trace/src/opentracing/span.js

```diff
@@ -20,6 +23,30 @@ const {
 const unfinishedRegistry = createRegistry('unfinished')
 const finishedRegistry = createRegistry('finished')
 
+const otelEnabled = !!process.env.DD_TRACE_OTEL_ENABLED
```

Comment:
How about `OTEL_ENABLED` since it's a top level constant?

Reply:
Done. :)